apps/myaccount/src/models/ua-parser.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable @typescript-eslint/interface-name-prefix */
+export interface IBrowser {
+    /**
+     * Possible values :
+     * Amaya, Android Browser, Arora, Avant, Baidu, Blazer, Bolt, Camino, Chimera, Chrome,
+     * Chromium, Comodo Dragon, Conkeror, Dillo, Dolphin, Doris, Edge, Epiphany, Fennec,
+     * Firebird, Firefox, Flock, GoBrowser, iCab, ICE Browser, IceApe, IceCat, IceDragon,
+     * Iceweasel, IE [Mobile], Iron, Jasmine, K-Meleon, Konqueror, Kindle, Links,
+     * Lunascape, Lynx, Maemo, Maxthon, Midori, Minimo, MIUI Browser, [Mobile] Safari,
+     * Mosaic, Mozilla, Netfront, Netscape, NetSurf, Nokia, OmniWeb, Opera [Mini/Mobi/Tablet],
+     * Phoenix, Polaris, QQBrowser, RockMelt, Silk, Skyfire, SeaMonkey, SlimBrowser, Swiftfox,
+     * Tizen, UCBrowser, Vivaldi, w3m, Yandex
+     *
+     */
+    name: string | undefined;
+
+    /**
+     * Determined dynamically
+     */
+    version: string | undefined;
+
+    /**
+     * Determined dynamically
+     * @deprecated
+     */
+    major: string | undefined;
+}
+
+export interface IDevice {
```

Comment:
Should we remove the "I" in the interface name?

Reply:
Well, this actually belongs to the `ua-parser` library. Since the library uses `declare` to define types and that is incompatible with `babel`, I had to move the type definitions into our codebase.